vm/devices/storage/disk_nvme/nvme_driver/src/driver.rs

```diff
@@ -399,8 +399,14 @@ impl<T: DeviceBacking> NvmeDriver<T> {
         let max_io_queues = allocated_io_queue_count.min(requested_io_queue_count);
 
         let qsize = {
-            let io_cqsize = QueuePair::MAX_CQ_ENTRIES.min(worker.registers.cap.mqes_z() + 1);
-            let io_sqsize = QueuePair::MAX_SQ_ENTRIES.min(worker.registers.cap.mqes_z() + 1);
+            if worker.registers.cap.mqes_z() < 1 {
+                anyhow::bail!("mqes cannot be 0");
+            }
+
+            println!("mqes value is {}", worker.registers.cap.mqes_z());
```

Comment:
I assume this is left over from debugging. If you think we should log this, please:

1. move it to a trace (`tracing::trace!`),
2. just log the entire contents of the `cap` register at this point.